jazz-build-module/service-configuration-data-loader.groovy

```diff
@@ -180,7 +213,7 @@ def loadServiceConfigurationData() {
 			sh "sed -i -- 's/{jazz_admin_creds}/${config_loader.JAZZ.PASSWD}/g' ./config/prod-config.json"
 		}
 
-		if ((service_name.trim() == "jazz_delete-serverless-service") || (service_name.trim() == "jazz_create-serverless-service")) {
+		if ((service_name.trim() == "jazz_delete-serverless-service") || (service_name.trim() == "jazz_create-serverless-service") || (service_name.trim() == "jazz_deployments")) {
```

Comment:
We may want to do something like this next time - 

if (['a', 'b', 'c'].indexOf(service_name.trim()) >= 0) {
    // do something
}